cmd/check_container.go

```diff
@@ -31,8 +36,9 @@ func checkContainerCmd() *cobra.Command {
 	checkContainerCmd.Flags().BoolVarP(&submit, "submit", "s", false, "submit check container results to red hat")
 	viper.BindPFlag("submit", checkContainerCmd.Flags().Lookup("submit"))
 
-	checkContainerCmd.Flags().String("pyxis-api-token", "", "API token for Pyxis authentication (env: PFLT_PYXIS_API_TOKEN)")
+	checkContainerCmd.Flags().StringVar(&pyxisApiToken, "pyxis-api-token", "", "API token for Pyxis authentication (env: PFLT_PYXIS_API_TOKEN)")
 	viper.BindPFlag("pyxis_api_token", checkContainerCmd.Flags().Lookup("pyxis-api-token"))
+	// checkContainerCmd.Flags().Lookup("pyxis-api-token").NoOptDefVal = "NOTSET"
```

Comment:
I think this line and line 53 might need to be removed or uncommented? Not sure which.

Reply:
Removed. Thanks for catching that. Added a test for going against a config file.